.github/workflows/main.yml

```diff
@@ -29,95 +31,71 @@ jobs:
-          sleep 30
-          
-      - name: Setting up kubeconfig ENV for Github Chaos Action
-        run: echo ::set-env name=KUBE_CONFIG_DATA::$(base64 -w 0 ~/.kube/config)
-
-      - name: Running Litmus pod delete chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-pod-delete') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: pod-delete
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true
-
-      - name: Running container kill chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-container-kill') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: container-kill
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          CONTAINER_KILL_HELPER_IMAGE: litmuschaos/container-kill-helper
-          CONTAINER_KILL_HELPER_TAG: latest
-          LIB: containerd
-          LITMUS_CLEANUP: true
-
-      - name: Running node-cpu-hog chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-node-cpu-hog') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: node-cpu-hog
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true
-
-      - name: Running node-memory-hog chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-node-memory-hog') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: node-memory-hog
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true      
-                      
+          CHOKIDAR_USEPOLLING=1 npm start &
+        working-directory: litmus-portal/frontend
+
+      - name: Installing Cypress and running tests for login page
+        if: startsWith(github.event.comment.body, '/run-e2e-login')
+        uses: cypress-io/github-action@v2
+        continue-on-error: false
+        with:
+          wait-on: http://localhost:3000
+          spec: cypress/integration/Login/*
+          working-directory: litmus-e2e/CypressE2E/
+
+      - name: Install Cypress and run all tests
+        if: startsWith(github.event.comment.body, '/run-e2e-all') || startsWith(github.event.comment.body, '/run-all')
```

Comment:
Do we need `run-all`? we can keep `run -e2e-all` for running all e2e tests. I can understand you've created `run-all` to differentiate between unit test so We can add them in `run-e2e` only.  Like:

`run-e2e-unit` - for unit test
`run-e2e-login` - for login test
`run-e2e-workflow` - for workflow test 
`run-e2e-all` - for all tests

Reply:
But what if we only want to run all e2e tests.
I used run-all to run all unit tests as well as e2e tests.